src/optional-listeners/message-command-listeners/CorePreMessageParser.ts

```diff
@@ -64,6 +67,7 @@ export class CoreListener extends Listener<typeof Events.PreMessageParsed> {
 		if (message.content[offset + id.length] !== '>') return null;
 
 		// Check whether or not the ID is the same as the managed role ID:
+		// TODO(vladfrangu): substr is marked as deprecated!!
```

Comment:
Make an issue or replace with `.substring` in the PR

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#the_difference_between_substring_and_substr